duplicati.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "duplicati",
+	"plugin_schema": "2",
+    "release": "11.3-RELEASE",
+    "artifact": "https://github.com/rexit1982/iocage-plugin-duplicati.git",
+    "properties": {
+        "dhcp": 1
+    },
+    "pkgs": [
+        "mono",
+        "py27-sqlite3",
+        "curl",
+		"ca_root_nss"
```

Comment:
Looks like another improper indentation snuck in

Reply:
Found some tabs instead of spaces, probably a copy/paste error.  Updated my fork, not sure if that automatically bring the updates into this PR or not, it seems to show my latest commit at the bottom.

Reply:
Yep, it get's automatically updated.